packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -60,55 +63,68 @@ function InterfaceSkeleton( {
 				'interface-interface-skeleton'
 			) }
 		>
-			{ !! header && (
-				<div
-					className="interface-interface-skeleton__header"
-					role="region"
-					aria-label={ mergedLabels.header }
-					tabIndex="-1"
-				>
-					{ header }
-				</div>
-			) }
-			<div className="interface-interface-skeleton__body">
-				{ !! leftSidebar && (
+			<div className="interface-interface-skeleton__layout">
```

Comment:
Previously, the navigation panel and navigation toggle lived within skeleton__body and skeleton__header, respectively. In order to address unexpected keyboard focus order between the nav toggle and nav panel, we create a new element inside of interface-skeleton called skeleton__navigation-sidebar.

To account for the layout changes of introducing a new element, we add two wrapper classes: skeleton__site and skeleton__content. Both ensure that the skeleton__header and skeleton__body shrink their children correctly when the navigation sidebar is open.